usl_pipeline/cloud_functions/requirements.txt

```diff
@@ -56,4 +57,5 @@ tzdata==2024.1
 urllib3==2.2.1
 watchdog==4.0.0
 Werkzeug==3.0.1
+wrf-python==1.3.4.1
```

Comment:
Noted elsewhere, but we'll need to wait to merge until we know how this impacts cloud functions, as this file is used to deployments right now.

Reply:
Ack